Inventory.spf

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's going on with this file?

Reply:
I have no idea when/how/where/if i created actually